gemfirexd/tools/src/main/java/com/pivotal/gemfirexd/tools/internal/GfxdServerLauncher.java

```diff
@@ -1255,24 +1255,22 @@ public void setWaitingStatus(String regionPath,
     StringBuilder otherMembers = new StringBuilder();
     String tableName = Misc.getFullTableNameFromRegionPath(regionPath);
     if (GemFireStore.DDL_STMTS_REGION.equals(tableName)) {
-      tableName = "DataDictionary";
+      tableName = "DataDictionary at location " + myId.directory;
```

Comment:
Please use a better name since you are also attaching the location dir of the table.